source/_layouts/guides/default.html

```diff
@@ -30,6 +30,14 @@ <h1 class="pagetitle">{{ page.title }}</h1>
         (Pebble Classic, Pebble Steel).
       </p>
     </div>
+    {% elsif page.platforms contains 'basalt' and page.platforms contains 'chalk' and page.platforms contains 'diorite' %}
+    <div class="alert alert--fg-white alert--bg-purple alert--medium">
+      <p>
+        <strong>PLATFORM NOTICE</strong><br>
+        This guide only applies for apps built to run on the Basalt, Chalk or Diorite platforms
+        (Pebble Time, Pebble Time Steel, Pebble Time Round, Pebble 2 and Pebble 2 SE).
```

Comment:
I wasn't sure if this is the best way to specify the difference between Pebble 2 and Core 2 Duo. It would be significantly easier if the entire documentation referred to Core 2 Duo as such.

Reply:
Are these any examples of pages that actually trip this? I don't expect there to be too many.

I'd also either say "Pebble 2 SE and Pebble 2 HR" or just "Pebble 2" here - leaning towards the latter (maybe with a "not Pebble 2 Duo" added?)

Reply:
Smart straps are the reason for this, since that's the one feature missing in c2d

Reply:
It also doesn't apply to aplite, so in either case you are aiming at 3 platforms with the message, but considering c2d, it might be easier to exclude it rather than include diorite watches